go.sum

```diff
@@ -25,6 +25,8 @@ github.com/creack/pty v1.1.9/go.mod h1:oKZEueFk5CKHvIhNR5MUki03XCEU+Q6VDXinZuGJ3
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/derekbit/longhorn-spdk-engine v0.0.0-20230614165804-a5ebfaf67c88 h1:xLyd+i/I5sUbNFy4owtP04XtI912by6czpJPg1OsVWk=
```

Comment:
Is it intentional that we are adding github.com/derekbit/longhorn-spdk-engine repo?

Reply:
Yeah, because https://github.com/longhorn/longhorn-spdk-engine/pull/53 is waiting for review.
If it is merged, the dependency will be corrected.